File: java/src/main/java/codingdojo/ErrorResult.java
package codingdojo;

public class ErrorResult {
    private final String formulaName;
    private final String message;
    private final String presentation;

    public ErrorResult(String formulaName, String message, String presentation) {

        this.formulaName = formulaName;
        this.message = message;
        this.presentation = presentation;
    }

    @Override
    public String toString() {
        return "ErrorResult{" +
                "formulaName='" + formulaName + '\'' +
                ", message='" + message + '\'' +
                ", presentation='" + presentation + '\'' +
                '}';
    }
}

File: csharp/ExceptionRollup/ErrorResult.cs
namespace codingdojo
{
    public class ErrorResult
    {
        private readonly string _formulaName;
        private readonly string _message;
        private readonly string _presentation;

        public ErrorResult(string formulaName, string message, string presentation)
        {
            _formulaName = formulaName;
            _message = message;
            _presentation = presentation;
        }


        public override string ToString()
        {
            return "ErrorResult{" +
                   "formulaName='" + _formulaName + '\'' +
                   ", message='" + _message + '\'' +
                   ", presentation='" + _presentation + '\'' +
                   '}';
        }

        public string GetFormulaName()
        {
            return _formulaName;
        }

        public string GetMessage()
        {
            return _message;
        }

        public string GetPresentation()
        {
            return _presentation;
        }
    }
}